### PR TITLE
Added missing-return-doc message example

### PR DESCRIPTION
**Pull request by matusvalo**
## Type of Changes

|     | Type                   |
| --- | ---------------------- |
|     | :bug: Bug fix          |
|     | :sparkles: New feature |
|     | :hammer: Refactoring   |
| ✓   | :scroll: Docs          |

## Description

Related #5953

### CODE REVIEW

File: doc/data/messages/m/missing-return-doc/good.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+def integer_sum(a, b):
+    """Returns sum of two integers
+    :param a: first integer
+    :type a: int
+    :param b: second integer
+    :type b: int
+    :return: sum of parameters a and b
+    :rtype: int
```
Comment (Pierre-Sassoulas) [inline doc/data/messages/m/missing-return-doc/good.py:8]:
```suggestion
def integer_sum(a, b) -> int:
    """Returns sum of two integers
    :param a: first integer
    :type a: int
    :param b: second integer
    :type b: int
    :return: sum of parameters a and b
```
I'm wondering if this does not make more sense to use the builtin typing directly.

Comment (matusvalo) [reply doc/data/messages/m/missing-return-doc/good.py:8]: > I'm wondering if this does not make more sense to use the builtin typing directly.

I stick closely to checker documentation https://pylint.pycqa.org/en/latest/technical_reference/extensions.html#pylint-extensions-docparams so I used comments. But I can rework it to annotation types...

Comment (matusvalo) [reply doc/data/messages/m/missing-return-doc/good.py:8]: I suppose than also other examples should be reworked to use python typing right?

Comment (Pierre-Sassoulas) [reply doc/data/messages/m/missing-return-doc/good.py:8]: I think this checker doc was done a long time ago (when 'real' typing was not available yet) possibly with python 2 compatibility in mind. The other example would need to be modified too but let's hear what @DanielNoord think before changing everything :)

Comment (matusvalo) [reply doc/data/messages/m/missing-return-doc/good.py:8]: I will update them no worries

**Pull request merged**